### ISSUE TITLE
Clear session on command ":exit"

### ISSUE THREAD

**Issue by BjoernLoetters** [morganey-lang/let-morganey-rest#16]
Instead of prohibiting the command ":exit" in the online repl, it would be nice to clear the session and display the welcome message again.

**Comment by keddelzz** [comment]
> Instead of prohibiting the command ":exit

linking this issue with #7

**Comment by keddelzz** [comment]
@rexim @ForNeVeR what do you think of clearing/resetting the session on command `:exit` in the online REPL?

**Comment by ForNeVeR** [comment]
I agree with the proposal.

**Comment by rexim** [comment]
Looks fine to me. It'd be also cool to clean the screen so it feels like a new session because the "previous one" was "exited". :)

**Comment by keddelzz** [comment]
I'll add this to the description.

**Comment by keddelzz** [comment]
You know what. I won't add it to the description, because this was suggested by @kuchenkruste.

**Comment by rexim** [comment]
> You know what. I won't add it to the description, because this was suggested by @kuchenkruste.

lol, I'm sorry! :D Looks like I have to be more careful while reading other people's comments.

**Comment by ForNeVeR** [comment]
I don't think it was suggested by @kuchenkruste yet. He says "clear the session", which is probably not the same as "clear the screen". I thought that clearing the session is the same as calling `:unbind`.

**Comment by keddelzz** [comment]
You're absolutely right: "clear the session" was suggested by @kuchenkruste and "clean the screen" by @rexim, but I meant, that I'll not edit the description of an issue, which was suggested by someone else than me.
